AUTHORS.txt

```diff
@@ -65,4 +65,5 @@ Robert Roos
 Sebastien Weber
 Sebastian Neusch
 Ulrich Sauter
-Guus Kuiper
\ No newline at end of file
+Guus Kuiper
+Armindo PINTO
```

Comment:
As you see from the other names, even the last name is in lower case (except for the first letter).
Unless your name is spelled that way officially, please adhere to the common standard.

Reply:
Hello, yes sorry for the TYPO. Change has been committed.